image/templates/helm/stackrox-central/internal/config-shape.yaml

```
@@ -94,6 +115,12 @@ customize:
     podLabels: {} # dict
     podAnnotations: {} # dict
     envVars: {} # dict
+  centralDB:
```

scanner's db is `scanner-db`. The helm chart is a bit inconsistent already, but let's try to keep it as consistent as possible here.

```suggestion
  central-db:
```

---

renamed to db: